### ISSUE TITLE
Go beyond sorting algorithm

### ISSUE THREAD

**Issue by Himanshu40** [Himanshu40/Sorting-Algorithm#4]
Thinking to add more algorithms rather than only sorting algorithm.

**Comment by Jaidev2001** [comment]
Can i add algorithms with c++ language?

**Comment by ApoorvaKaushal09** [comment]
Hey can I add the bucket sort code in java?

**Comment by Himanshu40** [comment]
> Hey can I add the bucket sort code in java?

Sure go for it.

**Comment by akarshan2602** [comment]
@Himanshu40 hey, can i add the cyclic sort code in java? Didn't see it there

**Comment by Himanshu40** [comment]
> @Himanshu40 hey, can i add the cyclic sort code in java? Didn't see it there

Yeah, sure. Go for it.

**Comment by akarshan2602** [comment]
Yeh thanks!

**Comment by akarshan2602** [comment]
will this be counted for hacktoberfest?? im new to these so idk much

**Comment by Himanshu40** [comment]
> will this be counted for hacktoberfest?? im new to these so idk much

Yeah, it will be counted towards hacktoberfest. Be sure to check README and CONTRIBUTING guide.

**Comment by akarshan2602** [comment]
Ok thanks!

**Comment by AnshuAyush** [comment]
Hey Can I include some searching algorithms like Binary, Linear, BFS,DFS

**Comment by Himanshu40** [comment]
> Hey Himanshu, can I add recursive approaches of Insertion and Bubble sort.

Yeah sure. Go for it. And make sure to name the file as ```RecursiveBubbleSort.extension``` where the extension can be any language.

**Comment by Himanshu40** [comment]
> > > Hey Can I include some searching algorithms like Binary, Linear, BFS,DFS
> > 
> > 
> > Yeah sure. But make sure to create a Search-Algorithm dir before making PR.
> 
> Hey its saying "Please make sure you have the correct access rights"

Yeah, you are cloning my repo instead of your forked repo. Make sure that you forked my repo and clone from your repo not from me.

**Comment by PrachiDadhich** [comment]
Hey can I implement Hashing for searching (Java)
